### PR TITLE
fix(curriculum): tests passing with seed code in Music player

### PR DESCRIPTION
**Pull request by gikf**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

- - -
- Fixes test passing with seed code.

### CODE REVIEW

**Review by moT01 — APPROVED** [review]
LGTM 🎉 

Just a side note, spaces are allowed between `.`'s.

**Comment by gikf** [comment]
> Just a side note, spaces are allowed between `.`'s.

Not sure how that's not coming up often on forums to be honest. Let me know if you'd want me to add them.

But surprise, between `?.` there cannot be space.

**Comment by moT01** [comment]
I think it's fine. If it's not coming up on the forum, it's probably not very common. Just thought I would mention it.

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/65a60aa3efd8fa079c2d1537.md [hunk 1]
```diff
@@ -33,10 +33,11 @@ Inside your callback function, add another `if` statement to check if `a.title`
 
 # --hints--
 
-You should have an `if` statement inside your callback function.
+You should have second `if` statement inside your callback function.
```
Comment (Sembauke) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/65a60aa3efd8fa079c2d1537.md:36]:
```suggestion
You should have a second `if` statement inside your callback function.
```

**Review by Sembauke — CHANGES_REQUESTED** [review]

**Review by Sembauke — APPROVED** [review]

**Pull request merged**